velox/docs/monthly-updates/december-2021.rst

```diff
@@ -0,0 +1,40 @@
+********************
+December 2021 Update
+********************
+
+Documentation
+-------------
+
+* Add :doc:`Aggregation-related optimizations <../develop/aggregations>` guide.
+* Add :doc:`json_extract_scalar <../functions/json>` function.
+* Update :doc:`Mathematical Functions <../functions/math>`.
+* Update :doc:`Comparison Functions <../functions/comparison>`.
+* Update :doc:`How to add an aggregate function? <../develop/aggregate-functions>` guide with new registry.
+* Update :doc:`How to add a scalar function? <../develop/scalar-functions>` guide with complex types.
```

Comment:
perhaps, change "with complex types" to "to describe how to write functions with complex types inputs and results".